Lesson 6 Task 1 - Printing File Content

jean-farinea opened the pull request:
1. Print class created with function to read and print file content.
2. Test class created to check function

jean-farinea committed:
1. Print class created with function to read and print file content.
2. Test class created to check function

strzeleckik commented on src/test/java/lesson6/task1/PrintTest.java:
```diff
@@ -0,0 +1,21 @@
+    @Test
+    void shouldPrintFileContent() throws IOException {
+        //given
+        String expectedContent = "Course\nJava\nLesson 6\nFiles\nExceptions\nFiles\nEnd of file";
```
If a run this test on my computer, test will fail, because, this file in somewhere outside project folder so you dont have it included in your repository. Am I right?

strzeleckik commented on src/main/java/lesson6/task1/Print.java:
```diff
@@ -0,0 +1,13 @@
+public class Print {
+
+    public static String printFileContent() throws IOException{
+        String content = FileUtils.readFromFile("src/main/resources/data.txt");
```
it would be better to move this path to a method parameter

strzeleckik reviewed (changes requested):
Few things to fix

jean-farinea committed:
1. Print class adjusted to get filePath as paramether input.
2. Test class adjusted to create filepath before running the test itself to make sure path and file exists.

jean-farinea commented:
Adjustments done. Please review.

strzeleckik reviewed (approved):
ok :)
